buildSrc/src/main/groovy/org/elasticsearch/gradle/test/RestIntegTestTask.groovy

```diff
@@ -162,7 +162,7 @@ public class RestIntegTestTask extends DefaultTask {
                 if (line.startsWith("[")) {
                     inExcerpt = false // clear with the next log message
                 }
-                if (line =~ /(\[WARN\])|(\[ERROR\])/) {
+                if (line =~ /(\[WARN *\])|(\[ERROR *\])/) {
```

Comment:
We don't *technically8 need the ` *` after `ERROR` but it feels like the right amount of paranoia.